api/placer.py

```diff
@@ -577,14 +578,18 @@ def finalize(self):
 
         # Extra properties on insert
         insert_map = copy.deepcopy(query)
-        insert_map.pop('subject.code', None) # Remove query term that should not become part of the payload
+
+        # Remove query term that should not become part of the payload
+        insert_map.pop('subject.code', None)
+        insert_map.pop('deleted')
+
         insert_map['created'] = self.timestamp
         insert_map.update(self.metadata['session'])
         insert_map['subject'] = containerutil.add_id_to_subject(insert_map.get('subject'), bson.ObjectId(self.p_id))
         if 'timestamp' in insert_map:
             insert_map['timestamp'] = dateutil.parser.parse(insert_map['timestamp'])
 
-        session = config.db['session' + 's'].find_one_and_update(
+        session = config.db.sessions.find_one_and_update(
```

Comment:
This is why they weren't found while updating all db calls to containers that handle delete